qemu/tests/cfg/blockdev_full_backup.cfg

```diff
@@ -56,10 +55,14 @@
                     only auto_compress
                     only auto_completed
                 - src_cluster_size_512:
+                    no nbd
+                    not_preprocess = yes
```

Comment:
we can remove this line for VT takes cares of image creation with cluster size option.
Do we want to add only qcow2 here?